recipe-server/client/control/state/action-types.js

```diff
@@ -0,0 +1,38 @@
+/* Keep this list alphabetized */
+
+export const ACTION_FETCH = 'ACTION_FETCH';
```

Comment:
In the past, I've done something like this to achieve the same effect, but with a little brevity:

```
['HI', 'THERE', 'ETC'].forEach((eventName)=>{
	module.exports[eventName] = eventName;
});

// then you can pull all of them as an object
import ActionTypes from 'path/to/action-types';
// or individually
import {ACTION_FETCH} from 'path/to/action-types';
```